doc/nrf/ug_esb.rst

```diff
@@ -72,6 +76,10 @@ ESB requires exclusive access to all fixed and configured resources for the :ref
 The radio and timer interrupt handlers run at priority level 0 (highest level), and the ESB callback functions run at priority level 1.
 Other interrupts used by the application must use priority level 2 or lower (level 2 to 7) to ensure correct operation.
 
+ESB requires the :ref:`MPSL library <nrfxlib:mpsl_lib>` for front-end module support.
+This library is always linked into the build, but it is not initialized by default to provide your application and the ESB protocol with access to all hardware resources.
```

Comment:
```suggestion
This library is always linked into the build, however, it must be initialized manually to provide your application and the ESB protocol with access to all hardware resources.
```

Reply:
I wrote it in a different way

Reply:
Ok, so maybe we can split this sentence into two?

**This library is always linked to the build, yet it is not initialized by default. As a result, your application and the ESB protocol can access all hardware resources.**